routes/web.js

```diff
@@ -10,36 +10,254 @@ import { body } from 'express-validator'
-router.get('/standups/responses', authMiddleware, standUpController.standUpResponses);
+
+/**
+ * @openapi
+ * /standups/subscribe:
+ *   post:
+ *     description: Subscribe to a standup
+ *     tags: [Standup]
+ *     parameters:
+ *       name: standup_id
+ *       $ref: '#/definitions/User'
+ *     responses:
+ *       200:
+ *         description: Return user
+ *         schema:
+ *           type: object
+ *           user:
+ *             $ref: '#/definitions/User'
+ *           success:
+ *              type: string
+ */
 router.post('/standups/subcribe', [
     body('standup_id').isString(),
     authMiddleware
 ], standUpController.subscribeToStandUp);
+
+/**
+ * @openapi
+ * /standups/unsubcribe:
+ *   post:
+ *     description: Unsubscribe to a standup
+ *     tags: [Standup]
+ *     parameters:
+ *       name: standup_id
+ *       $ref: '#/definitions/User'
+ *     responses:
+ *       200:
+ *         description: Return user
+ *         schema:
+ *           type: object
+ *           user:
+ *             $ref: '#/definitions/User'
+ *           success:
+ *              type: string
+ */
 router.post('/standups/unsubcribe', [
     body('standup_id').isString(),
     authMiddleware
 ], standUpController.unsubscribeToStandUp);
+
+/**
+ * @openapi
+ * /standups/update:
+ *   post:
+ *     description: Update a standup
+ *     tags: [Standup]
+ *     parameters:
+ *       name: standup_id
+ *       $ref: '#/definitions/Standup'
+ *     responses:
+ *       200:
+ *         description: Return standup
+ *         schema:
+ *           type: object
+ *           StandUp:
+ *             $ref: '#/definitions/Standup'
+ */
 router.post('/standups/update', [
     body('standup').isObject(),
     authMiddleware
 ], standUpController.updateStandUp);
+
+/**
+ * @openapi
+ * /standups/update:
+ *   post:
+ *     description: Complete standup update
+ *     tags: [Standup]
+ *     parameters:
+ *       $ref: '#/definitions/StandupUpdate'
+ *     responses:
+ *       200:
+ *         description: Return standup update
+ *         schema:
+ *           type: object
+ *           StandUp:
+ *             $ref: '#/definitions/StandupUpdate'
+ */
 router.post('/standups/complete', [
     body('standup_update').isObject(),
     authMiddleware
 ], standUpController.completeStandUp);
+
+/**
+ * @openapi
+ * /users:
+ *   get:
+ *     description: Get users
+ *     tags: [User]
+ *     responses:
+ *       200:
+ *         description: Return users
+ *         schema:
+ *           type: object
+ *           users:
+ *             $ref: '#/definitions/User'
+ */
 router.get('/users', authMiddleware, userController.users);
 
 //Authentication routes
 
+/**
+ * @openapi
+ * definitions:
+ *   Login:
+ *     required:
+ *       - username
+ *       - password
+ *     properties:
+ *       username:
+ *         type: string
+ *       password:
+ *         type: string
+ *       path:
+ *         type: string
+ */
+
+/**
+ * @openapi
```

Comment:
Can you put this definition along side the other ones in swaggerJSDocOptions?

Reply:
Sure thing!